hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+
+## 5.1. The Signal Generator settings
+At beginning of the test, it should configure SA to generate right modulated signal. Following steps will configure a GFSK modulated signal:
+
+- Set the SG to Custom Mode;
+- Set IQ data to PN9;
+- Set modulation type to 2-level FSK;
+- Set filter to 0.5Bt Gaussian type;
+- Set symble rate to the desired data rate, symble rate=data rate for 2GFSK modulation;
+- Set frequency deviation to desired frequency deviation; 
+- Set frequency to desired operating frequency;
+- RF: ON
+- MOD: ON
+- Tune signal strength level;
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\BER3.png">  
+</div>
+
+## 5.2. RailTest commands and test result
+The RailTest commands are listed below for Rx BER test: 
+  - :>rx 0
+  - :>Setchannel 0 
+  - :>SetBerconfig 1000
+  - :>BerRx 1
+  - :>......  
+  - :>Berstatus
+
+Railtest commands explaination:
+- Railtest starts with "rx 0" command;
+- Setchannel 0 will select a frequency points to test, it can also select another frequency point to test by changing the channel number;
+- SetBerconfig 1000 sets 1000 bytes(8000 bits) data to count the BER;
+- BerRx 1 sets the radio in BER RX mode;
+- Wait for a while. Then the Berstatus command will display the BER test result.
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\BER4.png">  
+</div>
+
+When the PercentBitError rise up to 0.10, the SA's signal strength minus cable loss is the sensitivity 
+under 0.1% BER.
+
+# 6. Radiation test environment
+Radiation test should be performed in a RF shielded anechoic chamber. Most of the regulation test items can be done with radiation test in an anechoic chamber. Besides that some antenna radiation parameter can be tested with radiation test such as TRP(Total Radiation Power), TIS(Total Isotropic Sensitivity), 2D cuts, and 3D pattern.
+## 6.1. Typical radiation test chamber SAR 
+Generally there are 3 types of test site(environment) used to do radiation test:
+- Open Area Test Site (OATS); 
+- Semi Anechoic Room (SAR); 
+- Fully Anechoic Room (FAR). 
+They are referred to as free field test sites. Both absolute and relative measurements can be performed on these test sites.
+Most commonly used site is SAR for radiation test. The following picture shows the site configuration:
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\SAR-Site.png">  
+</div>
+
+A Semi Anechoic Room is anechoic chamber with a conductive ground plane, and is an enclosure, usually shielded, 
+whose internal walls and ceiling are covered with radio absorbing material. The floor, which is metallic, is not covered by absorbing material and forms the ground plane.The chamber usually contains an antenna mast at one end and a turntable at the other end.
```

Comment:
Insert space to change "plane.The" to "plane. The".

Reply:
Corrected.